### PR TITLE
Edit tasks fixed

### PR DESCRIPTION
**Pull request by Raisa-12**
### Functionality:
the assignee's email and the picture are not updated immediately after we edit the task. Fix it.

### Solution:
made changes in tasks-details.component.ts

### Risk level:
- [ ] high 
- [X] medium
- [ ] low

### How to test:
Go to edit task and change the assignee to see the results

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -136,5 +136,8 @@
       }
     }
   },
-  "defaultProject": "worktrolly-ui"
+  "defaultProject": "worktrolly-ui",
+  "cli": {
+    "analytics": "e80cdbbd-7976-468d-b648-98a34814b487"
+  }
 }
```
Comment (viv9k) [inline angular.json:143]: revert this change

File: src/app/body/task-details/task-details.component.ts [hunk 2]
```diff
@@ -81,15 +81,6 @@ export class TaskDetailsComponent implements OnInit {
...
   prApiLink: string;
-  totalEstimatedTime: number;
-  estimatedTimeHrs: number;
-  estimatedTimeMins: number;
-  totalLoggedTime: number;
-  loggedTimeHrs: number;
-  loggedTimeMins: number;
-  totalRemainingTime: number;
-  remainingTimeHrs: number;
-  remainingTimeMins: number
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:92]: why remove this

File: src/app/body/task-details/task-details.component.ts [hunk 3]
```diff
@@ -185,20 +175,7 @@ export class TaskDetailsComponent implements OnInit {
...
-    [this.loggedTimeHrs, this.loggedTimeMins] = this.toolsService.changeToHourMinsTime(this.totalLoggedTime)
-
-    this.totalRemainingTime= this.totalEstimatedTime - this.task.LogWorkTotalTime;
-    [this.remainingTimeHrs, this.remainingTimeMins] = this.toolsService.changeToHourMinsTime(this.totalRemainingTime)
-    
-    if(this.remainingTimeHrs==0 && this.remainingTimeMins==0){
-      this.totalRemainingTime=0
-    }
-  }
+
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:178]: why remove so many things

File: src/app/body/task-details/task-details.component.ts [hunk 4]
```diff
@@ -244,25 +221,6 @@ export class TaskDetailsComponent implements OnInit {
...
-      error: (error) => {
-        this.errorHandlerService.showError = true;
-        this.errorHandlerService.getErrorCode(this.componentName, "InternalError", "Api");
-        console.error(error);
-      },
-      complete: () => {
-        this.getLinkData()
-        this.getTaskDetail()
-        console.info('Successfully created Link')
-
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:262]: Why are we removing this data

**Review by viv9k — APPROVED** [review]
Approved

**Review by Pendyala-thanvi — APPROVED** [review]
